README.md

```diff
@@ -1,22 +1,103 @@
+     - `.NET`
+     - `.Net Multi-Platform App UI`
+     - `Android`
+     - `iOS`
+   - You do **not** need to sign into Visual Studio, but the installation will require your macOS password.
+   - Once the installation is complete and you see the option to create or open a project, simply close the program. We won’t need to open Visual Studio after this.
+
+## Installing Celeste (FNA Linux Build)
+
+### 1. **From itch.io**  
+   - You can download/buy Celeste directly [here](https://mattmakesgames.itch.io/celeste).  
+   - After purchase or login, download the **Celeste Linux** Version.
+
+### 2. **From Steam**  
+   - Make sure Steam is installed on your system.
+   - Make sure Steam is **not** open.
+   - Type `steam://open/console` into the URL bar of your browser, and allow it to open Steam.
+   - Steam should open with a new **Console** Tab, go into it.
+   - Type `download_depot 504230 504233 1505052356460012099` into the Steam console and wait for it to install (there won't be a progress bar).
+   - Once complete, it will tell you where the files were installed. It will say something like this:
+   
+     ```bash
+     Depot download complete: "/Users/username/Library/Application Support/Steam/Steam.AppBundle/Steam/Contents/MacOS\\steamapps\\content\\app_504230\\depot_504233" (1259 files, manifest 1505052356460012099)
+     ```
+
+   - In this case, `depot_504233` is your Celeste folder that you should give to the build script. *This may vary.*
+   - Please move this folder to somewhere else, like your Downloads, or Documents folder, you can also rename it something like `Celeste-Linux`
+
+Once you have installed Celeste, make sure the folder contains content such as `Celeste.exe`, `Content` (folder), `Backups` (folder), `lib` (folder), `lib64` (folder).
+
+Also, Make sure the Name of this folder has no spaces in it, and make sure the path it's in has no spaces in it.
+
+
+## Building the IPA
+
+1. Open Terminal and navigate to where you'd like to install the build:
+   ```bash
+   cd Documents  # or your preferred directory
+   ```
+
+2. Clone the repository:
+   ```bash
+   git clone https://github.com/RoootTheFox/celeste-ios.git
+   ```
+
+      ****Make sure the path you clone into doesn't have any spaces in it, as this can cause problems***
+
+3. Navigate to the cloned directory:
+   ```bash
+   cd celeste-ios
+   ```
+
+4. Run the build script:
+   ```bash
+   ./build.sh
+   ```
+
+5. When prompted, drag and drop your celeste folder you downloaded earlier into the Terminal and press **Enter**.
+
+6. The build script will then ask:
+   - **"Do you want to enable a virtual game controller if no other input devices are detected? iOS 15+ only! (y/n)"**  
+     My recommendation is to type `n` for **no** unless you plan on using touch controls, as there could be issues with iOS 16 and above not auto-disabling the on-screen controller when a physical controller is connected.
```

Comment:
this sounds like a bug?
i was considering removing this prompt entirely (and just always having it build touch controller support) - can you tell me more about the mentioned issues?

Reply:
In iOS 17, it worked perfectly, if when launching the game you had a controller connected there would be no on screen controls, however, after updating to iOS 18, I couldn't get the controls to go away, even if freshly opening the game while a controller was connected, I haven't opened an issue about it because my certs have now run out, so I don't currently have it installed on my phone, so can't give details or anything, so defo needs more testing